tf/examples/stage/odc_k8s_core.tf

```diff
@@ -0,0 +1,25 @@
+module "odc_k8s_roles" {
+  # source = "github.com/opendatacube/datacube-k8s-eks//odc_eks?ref=terraform-aws-odc"
+  source = "../../odc_k8s_roles"
+
+  region = "ap-southeast-2"
+  owner = "odc-test"
+  namespace = "odc-test"
+  environment = "stage"
+
+  domain_name = "test.dea.ga.gov.au"
+
+  cluster_autoscaler_enabled = true
+  alb_ingress_enabled        = true
+}
```

Comment:
rather than a module that does all roles, could this be a module for making a generic role? 

name + policy = role

Reply:
good idea. will do refactoring after a break.